### DISCUSSION TITLE
SRV record config on Deadwood

### DISCUSSION THREAD

**Discussion by cingolo** [samboy/MaraDNS#127] [Q&A]
Hi Team,
Can someone confirm if Deadwood support SRV records? I didn't find anything into the documentation other than a way to configure the SRV record on MaraDNS so I m assuming that only MaraDNS supports SRV records. My goal is to resolve SRV record and A records on my windows workstation. 
Some how MaraDNS doesn't work any more on my Windows 11 workstation. it used to work after the fresh installation but then restarting window and trying to launch from dos ( administraotor) I m getting the window blue pop us saying that this program doesn't work on my laptop.
Thanks
Andrea

**Comment by samboy** [comment]
Deadwood caches SRV records and any other record types (except records from 65392 and 65407 which are “private use” DNS records) as long as they fit in the classic 512-byte DNS packets.

I do not have Windows 11 so have not had a chance to troubleshoot those issues. The workaround is to run it in Linux — I believe Microsoft now makes official Linux containers which can run in Windows.
